integration-tests/ssr/package.json

```diff
@@ -40,7 +40,9 @@
     "develop": "cross-env GATSBY_EXPERIMENTAL_DEV_SSR=true CI=1 FORCE_COLOR=0 gatsby develop",
     "serve": "gatsby serve",
     "start-dev-server": "start-server-and-test develop http://localhost:8000 test:jest",
-    "test": "cross-env GATSBY_EXPERIMENTAL_DEV_SSR=true npm-run-all -s build start-dev-server",
+    "capture-prod-dsg-and-ssr": "start-server-and-test serve http://localhost:9000 write-dsg-and-ssr-to-files",
+    "write-dsg-and-ssr-to-files": "node write-dsg-and-ssr-to-files.js",
```

Comment:
SSR page is not written to `public` (that's good :) ) but we need to have rawhtml from it, so those helpers run `gatsby serve` after the build and fetch dsg/ssr pages and save it to fs (gitignored `<rootOfSite>/prod-engines-output/<something>.html`) to compare later with DEV_SSR version of those pages.

For now we have single non-error SSR page to assert it works.